### ISSUE TITLE
Rendering Greek letters when no math is needed

### ISSUE THREAD

**Issue by janicemhdc** [asciidoctor/asciidoctor#3451]
I need to specify one Greek letter in my current documentation being produced in both HTML and PDF. I went searching the docs for how to do this and didn't see it specifically addressed. The docs do specifically say not to use HTML character encodings for any special characters if you're converting to PDF, so &foo; or &xxx; are out. The only option I see is to turn on the STEM/MathJax features and use them. This is working but it seems like major overkill to me as I do not need anything other than this single Greek letter - is there an easier way to do this?

**Comment by Mogztter** [comment]
Not sure if it's working in Asciidoctor PDF but what about `pass:[&Delta;]` ?

**Comment by Mogztter** [comment]
According to Dan:

> Asciidoctor PDF only supports decimal character references. But you can reference any character that's supported by the font.

https://github.com/asciidoctor/asciidoctor-pdf/issues/969#issuecomment-454630006

So if you're using the default theme and if Noto Serif supports your greek letter then you can use the decimal character reference:

`&#916;`: &#916;

**Comment by janicemhdc** [comment]
Okay, thanks.

I'll look for that spot in the documentation again and see if it should/could be clarified. I may also add a short note specifically about using this for Greek letters later when I have more time (I'm on deadline right now) as it would be nice if searching for "greek" yielded something useful.

It never occurred to me to put the issue under asciidoctor-pdf because I was asking for general syntax that would work everywhere, but in retrospect I can see why you think it might belong there. I admit I sometimes find the multitude of subprojects difficult to navigate.

**Comment by Mogztter** [comment]
> It never occurred to me to put the issue under asciidoctor-pdf because I was asking for general syntax that would work everywhere, but in retrospect I can see why you think it might belong there.

You're right the syntax should not depend on the backend (at least in theory).
In this case, I feel like this issue is about a "limitation" of Asciidoctor PDF (or at least a restricted feature) that's why I was suggesting to move it over there.

It's worth noting that using HTML entities should work with Asciidoctor PDF (JavaScript) because we generate an HTML page to produce a PDF document: https://github.com/Mogztter/asciidoctor-pdf.js

> I admit I sometimes find the multitude of subprojects difficult to navigate.

No worries 😉 
Asciidoctor is a wide ecosystem!

**Comment by mojavelinux** [comment]
> Asciidoctor PDF only supports decimal character references. But you can reference any character that's supported by the font.

That's outdated. Asciidoctor PDF supports all character references (named, decimal, and hexidecimal). The only limitation is that it only supports a limited set of named entities. Using decimal or hexidecimal is recommended. The other choice is to just paste the character itself.

**Comment by mojavelinux** [comment]
> I was asking for general syntax that would work everywhere

The official syntax should work everywhere. If/when it doesn't, we should consider that as a deficiency and address it.

**Comment by mojavelinux** [comment]
@janicemhdc Does using the decimal character reference, such as `&#916;` for delta, fit your requirements? If not, is there another syntax you would expect to work?

**Comment by mojavelinux** [comment]
The theming guide now states:

> All decimal and hexadecimal character references are supported, as well as the named entities amp, apos, gt, lt, nbsp, and quot (e.g., `&apos;`).

**Comment by janicemhdc** [comment]
Yes, I believe that's fine.

I generally look in any or all of the quick start, writer's guide, and user's guide until I find what I need. I wouldn't expect to find information about how to specify content in a theming guide - should I be looking for information there too? I'm using the default theme so I've not paid attention to it.

I do not currently see anything that indicates the decimal references shouldn't work so I'm not sure why I thought that other than I'm fried because of my deadline. I think I must have misread the note below which appears in both the quick reference and the user guide.

```
 Asciidoctor allows you to use any of the named character references (aka named entities) defined in HTML (e.g., &euro; resolves to €). However, using named character references can cause problems when generating non-HTML output such as PDF because the lookup table needed to resolve these names may not be defined. Our recommendation is avoid using named character references—​with the exception of those defined in XML (i.e., lt, gt, amp, quot and apos). Instead, use numeric character references (e.g., &#8364;).```

**Comment by mojavelinux** [comment]
>
>
> I generally look in any or all of the quick start, writer's guide, and
> user's guide until I find what I need. I wouldn't expect to find
> information about how to specify content in a theming guide - should I be
> looking for information there too?
>
No, not necessarily. I was letting Guillaume know that the exception that
was in that guide was out of date and I updated it.

The user manual should make it clear that character references are always
supported, so I'll make that update too. (This will be a clearer once the
new docs site is up).
